src/wp-includes/blocks.php

```diff
@@ -835,11 +834,12 @@ function make_before_block_visitor( $context ) {
  * @since 6.4.0
  * @access private
  *
- * @param WP_Block_Template|array $context A block template, template part, or pattern that the blocks belong to.
+ * @param WP_Block_Template|array $context     A block template, template part, or pattern that the blocks belong to.
+ * @param array                   $block_hooks An array of blocks hooked to another block.
  * @return callable A function that returns the serialized markup for the given block,
  *                  including the markup for any hooked blocks after it.
  */
-function make_after_block_visitor( $context ) {
+function make_after_block_visitor( $context, $block_hooks ) {
```

Comment:
(See https://github.com/WordPress/wordpress-develop/pull/5399/files#r1350123543 and https://github.com/WordPress/wordpress-develop/pull/5399/files#r1350126219.)

Reply:
Addressed with https://github.com/WordPress/wordpress-develop/pull/5399/commits/21efb9edb6f4ef1881cdb6bed8fa1c97c65ba0f6.